project/Dependencies.scala

```diff
@@ -19,7 +19,7 @@ object Dependencies {
   val Versions = Seq(
     crossScalaVersions := Seq("2.11.8"), // "2.12.0"
```

Comment:
is that the last thing needed for using `crossScalaVersions := Seq("2.11.8", "2.12.0")
does our release script already do `+ publishSigned`